main/SQLHelper.cpp

```diff
@@ -2967,7 +2967,8 @@ std::vector<std::vector<std::string> > CSQLHelper::query(const std::string &szQu
 
 	if (_log.isTraceEnable()) {
 		_log.Log(LOG_TRACE, "SQLQ query : %s", szQuery.c_str());
-		if (!_log.TestFilter("SQLR"))
+		char tmp[5]="SQLR";
+		if (!_log.TestFilter(tmp))
```

Comment:
Because of this:
[ 22%] Building CXX object CMakeFiles/domoticz.dir/main/SQLHelper.cpp.o
/home/pi/domoticz/main/SQLHelper.cpp: In member function ‘std::vector<std::vector<std::basic_string<char> > > CSQLHelper::query(const string&)’:
/home/pi/domoticz/main/SQLHelper.cpp:2970:30: warning: deprecated conversion from string constant to ‘char*’ [-Wwrite-strings]
   if (!_log.TestFilter("SQLR"))